### PR TITLE
2370 update reminders

### LINKED ISSUE #2370: Review and Revision Reminders
**Describe the bug**
- [x] Reminders are sent to the wrong account
- [x] The interface for this is not useful
- [x] Add a separate reminder for accepted reviews

### PR DESCRIPTION
**Pull request by ajrbyers**
- Makes the interface for managing reminders better
- Adds a new reminder type for accepted reviews
- Fixes a bug where the wrong recipient was being selected

Closes #2370

### CODE REVIEW

File: src/cron/views.py [hunk 1]
```diff
@@ -32,61 +32,62 @@ def reminders_index(request):
...
         'reminders': reminders,
-        'form': form,
     }
 
     return render(request, template, context)
 
 
 @editor_user_required
-def edit_reminder(request, reminder_id):
+def edit_reminder(request, reminder_id=None):
```
Comment (mauromsl) [inline src/cron/views.py:50]: This change means one can edit a reminder without passing through a reminder_id but that doesn't make a lot of sense. Is this change intended? Are you allowing creation of reminders here? Worth updating the name and the docstring in that case.

**Review by mauromsl — COMMENTED** [review]
Looks good, a few comments inline

File: src/templates/admin/cron/reminders.html [hunk 2]
```diff
@@ -13,59 +13,63 @@
...
                 </form>
             </div>
         </div>
     </div>
 {% endblock body %}
 
 {% block js %}
-    {% include "elements/datepicker.html" with target="#id_start_display" %}
-    {% include "elements/datepicker.html" with target="#id_end_display" %}
+{% include "admin/elements/post_check.html" %}
```
[the remark this answers is not in the document]
Comment (ajrbyers) [reply src/templates/admin/cron/reminders.html:74]: The delete button does have the class delete though.

`button type="submit" name="delete" value="{{ reminder.pk }}" class="tiny alert button delete"`

**Comment by ajrbyers** [comment]
@mauromsl ready for review again.

**Review by mauromsl — APPROVED** [review]
